packages/account-balances/src/balanceUpdates.ts

```diff
@@ -0,0 +1,13 @@
+import { Amount, Address, TxRefStr as TxRefBase, JsonObject } from "@proximaone/stream-schema-base";
+
+export type TxRef = Pick<TxRefBase, "blockNumber" | "blockHash" | "txHash">;
+
+export interface AccountBalanceUpdateEvent extends JsonObject {
+  chainId: string;
+  tokenId: string;
+  account: Address;
+  currentAmount: Amount;
+  // The last transaction due to which the balance has changed.
+  // It may be an uncled transaction if a rollback happens.
+  lastChange?: TxRef;
```

Comment:
very good. it allows us to build an api like `get me balance of account X at block N`